observation_portal/accounts/signals/handlers.py

```diff
@@ -0,0 +1,25 @@
+import json
+from django.dispatch import receiver
+from django.db.models.signals import post_save
+from django.contrib.auth.models import User
+from django.utils.module_loading import import_string
+from django.conf import settings
+
+from observation_portal.accounts.tasks import update_or_create_client_applications_user
+from observation_portal.accounts.models import Profile
+
+
+@receiver(post_save, sender=User)
+def cb_user_post_save(sender, instance, created, *args, **kwargs):
+    # Only update the user if it was not just created. If it was just created, we must wait until
+    # the Profile is created to send the update since we need profile information, which should happen
+    # immediately after user creation in our registration system.
+    if not created and not (kwargs.get('update_fields') == frozenset({'last_login'})):
```

Comment:
updates to only 'last_login' are filtered out since they occur on admin interface login, and are not part of the profile API so they won't be propagated to the client apps users.